README.md

```diff
@@ -31,14 +31,13 @@ and set your `PYTHONPATH` so that it includes the `eessi/reframe` directory from
 ```
 module load ReFrame/4.0.1
 
-eessiroot=<path_to_test-suite>
-eessihome=$eessiroot/eessi/reframe
+export PYTHONPATH=/path/to/EESSI/test-suite:$PYTHONPATH
 
-PYTHONPATH=$PYTHONPATH:$EBROOTREFRAME:$eessihome reframe \
-    -C <path_to_site_config_file> \
-    -c $eessihome/eessi_checks/applications/ \
-    -t CI -t 1_node \
-    -r --performance-report
+reframe \
+    --config-file <path_to_site_config_file> \
+    --checkpath eessi/testsuite/tests/apps \
```

Comment:
this assumes that you are in the `test-suite` directory, which is not ideal:
- logfiles and reframe test output will clutter the directory
- if you pip-installed it, you have to go look for this directory in ~/.local

to get the pip-installed dir you could do something like
```
testsuite=$(python3 -c 'import importlib; print(importlib.util.find_spec("eessi.testsuite").submodule_search_locations[0])')
```

Reply:
But then we need to discriminate between an installation with `pip`, and one without `pip` (on top of a checkout of this repo, for example)...

These instructions assume you're running from a checkout, so they're good enough, we can add a separate section on running when installed via `pip` later?
Or maybe we can somehow make sure that the path to the tests is considered automatically by ReFrame, not sure if that's possible...

Is this worth blocking this PR over?

Reply:
adding a separate section for a pip install sound good, we can do that in another PR.

can you add a note that we are indeed assuming you're running from a checkout, to make this clear?